0.10.4/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM debian:latest
```

Comment:
I'd move this Dockerfile to the root, since we're only going to maintain one version right now.